src/encoding.ts

```diff
@@ -75,6 +75,11 @@ export interface Encoding {
    * stack order for stacked marks or order of data points in line marks.
    */
   order?: OrderFieldDef | OrderFieldDef[];
+
+  /**
+   * Tooltip adds basic string tooltip support.
+   */
+  tooltip?: FieldDef | ValueDef<string>;
```

Comment:
I think this can only be TextFieldDef<F> | ConditionalValueDef<string>
also moved above order.